packages/office-ui-fabric-react/src/components/Shimmer/examples/Shimmer.Application.Example.tsx

```diff
@@ -62,13 +62,16 @@ export class ShimmerApplicationExample extends BaseComponent<{}, IShimmerApplica
         />
         <div>
           <ShimmeredDetailsList
-            setKey="items"
-            items={items!}
-            columns={columns}
-            selectionMode={SelectionMode.none}
-            onRenderItemColumn={this._onRenderItemColumn}
             enableShimmer={!isDataLoaded}
-            listProps={{ renderedWindowsAhead: 0, renderedWindowsBehind: 0 }}
+            items={items} // This will be deprecated in Fabric 7.0. Use the `detailsListProps` like bellow to pass the items.
+            detailsListProps={{
```

Comment:
Forcing everything to go through `detailsListProps` kind of breaks the point of the decorator. `ShimmeredDetailsList` should be interchangeable with `DetailsList` and support the same props as input.

Reply:
`ShimmeredDetailsList` has a `styles` prop that uses `IShimmeredDetailsListStyleProps` and `IShimmeredDetailsListStyles` which makes impossible to pass trough the DetailsList's `styles`. This regression was introduced during the conversion to CSS-in-JS and I am trying to fix it for the current major having backward compat and right away have a breaking change to fix it in Fabric 7 branch. I will make some changes to this PR into trying to make it more clear of the strategy for a good fix. Thanks!